server/api/core/email.js

```diff
@@ -0,0 +1,59 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { Job } from "meteor/vsivsi:job-collection";
+import { Jobs, Packages, Templates } from "/lib/collections";
```

Comment:
import { Assets } from "/lib/collections"; 
The missing import gives me an error